.github/workflows/publish.yml

```diff
@@ -20,11 +20,16 @@ jobs:
         with:
           persist-credentials: false
 
+      - name: Remove hardcoded MSRV
+        # Okay to build examples using the latest Rust.
+        # Fixes issues building wasm-bindgen-cli
+        run: rm -f rust-toolchain
+
       - name: Install Rust WASM target
         run: rustup target add wasm32-unknown-unknown
 
       - name: Install wasm-bindgen-cli
-        run: cargo install wasm-bindgen-cli --version=0.2.83
+        run: cargo install wasm-bindgen-cli --version=0.2.84
```

Comment:
Note that this was also updated to match our wasm-bindgen version:
https://github.com/gfx-rs/wgpu/blob/3432aabab158b2183d4eb2b6f58b74639acad2f3/Cargo.toml#L120